src/Controller/Game/BaseGameController.php

```diff
@@ -28,7 +28,7 @@ public function getGameUser(): User
             throw new AccessDeniedException('User is not enabled!');
         }
 
-        if ($user->getActive() !== true) {
+        if ($checkActive && $user->getActive() !== true) {
             throw new AccessDeniedException('User is not active!');
```

Comment:
As you make use of the Symfony Security component i suggest taking a look at the [User Checker](https://symfony.com/doc/current/security/user_checkers.html) to perform extra checks (like "is the user banned") on the account during authorization instead of on the controller level.

Reply:
I will look into the firewall component in combination with the User Checker to solve this later. Users should be able to login now, even though they are banned. Users can fill in a form to request an unban.